### PR TITLE
Refactor common fixtures to use kustomize

### PR DESCRIPTION
**Pull request by ishustava**
This reduces the duplication between fixtures and also makes it easier to add new fixtures that we will need for namespace tests.

### CODE REVIEW

**Review by thisisnotashwin — APPROVED** [review]
💥

File: test/acceptance/tests/fixtures/cases/static-client-inject/patch.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: static-client
+spec:
+  selector:
+    matchLabels:
+      app: static-client
```
Comment (lkysow) [inline test/acceptance/tests/fixtures/cases/static-client-inject/patch.yaml:8]: Do you need these lines? I briefly tested without them and it generated the same?

**Review by lkysow — APPROVED** [review]

File: test/acceptance/tests/fixtures/cases/static-client-inject/patch.yaml (hunk 1 above)
Comment (ishustava) [reply test/acceptance/tests/fixtures/cases/static-client-inject/patch.yaml:8]: Interesting,  I thought I did but that could have been one of many variations I tried that I don't remember now 😄 Let me try.

Comment (ishustava) [reply test/acceptance/tests/fixtures/cases/static-client-inject/patch.yaml]: Yup, didn't need those, thanks!

**Pull request merged**